indico/util/string.py

```diff
@@ -550,3 +550,10 @@ def __getstate__(self):
     def __setstate__(self, state):
         for slot, value in state.iteritems():
             setattr(self, slot, value)
+
+
+class MarkdownText(Markup):
```

Comment:
check if it's picklable. `Markup` defines an empty `__slots__` and i'm not sure if pickle only fails if `__slots__` is not empty or whenever it's defined